modules/tinymce/changelog.txt

```diff
@@ -6,6 +6,7 @@ Version 5.4.0 (TBD)
     Fixed `fullscreen` plugin now removes all classes when the editor is closed #TINY-4048
     Fixed handling of mixed-case icon names by having the silver theme downcase them before icon retrieval #TINY-3854
     Fixed an issue where removing formatting in a table cell would cause Internet Explorer 11 to scroll to the end of the table #TINY-6049
+    Fixed `autolink` no longer mistakes a string with multiple "@" as an email address.
```

Comment:
Suggestion:

``
Fixed the `autolink` feature so that it no longer treats a string with multiple "@" characters as an email address.
```

Reply:
This is also missing the JIRA number, which is required.

Reply:
Woups